packages/vue-app/template/client.js

```diff
@@ -150,9 +145,7 @@ async function loadAsyncComponents(to, from, next) {
     // Call next()
     next()
   } catch (err) {
-    const error = err || {}
-    const statusCode = (error.statusCode || error.status || (error.response && error.response.status) || 500)
-    this.error({ statusCode, message: error.message })
+    this.error(err)
     this.<%= globals.nuxt %>.$emit('routeChanged', to, from, error)
```

Comment:
This change causes:
```
ReferenceError: Can't find variable: error
```

Reply:
Could you create a new issue with a reproduction repo for that?

Reply:
If line 153 used to define `error`, which is then removed in this PR, but line 156/149 still references to this `error` variable, it can only go wrong.

I'll try to reproduce and create a new issue with minimal reproduction repo.

Reply:
Oh, I see, you are right @jorisbontje 🙈 
Would you submit a PR?

Reply:
@manniL @jorisbontje https://github.com/nuxt/nuxt.js/pull/4442

Reply:
Nvm, already there ☺️ 
https://github.com/nuxt/nuxt.js/pull/4442

Thanks for the info!